addons/safestart/functions/pfh_exit.sqf

```diff
@@ -23,7 +23,7 @@ if !(isNil "ace_interaction_fnc_showMouseHint") then {
 };
 // Reenable player damage, ACE throwing.
 player allowDamage true;
-ace_advanced_throwing_enabled = true;
+ace_advanced_throwing_enabled = GVAR(ace_throwing);
```

Comment:
If not running ace this will be nil, suggest `missionNamespace getVariable [QGVAR(ace_throwing), true];` to cater for case where it has not been set.

Reply:
I'll just wrap it with an isClass check instead, leaving it as nil.